public/class-public.php

```diff
@@ -149,9 +163,53 @@ public function new_account_page_content() { ?>
     </div>
     <?php
   }
-}
+  public function customer_order_form_submit(){
+    if (isset($_POST['customer']) && $_POST['customer'] != null) {
+      setcookie("GIGFILLIATE_PLACING_ORDER_FOR_CUSTOMER", $_POST['customer'], time() + (86400 * 30), "/");
+    }
+    header('Location: ' . $_SERVER['HTTP_REFERER']);
+  }
+  public function exit_customer_form_submit(){
+    if (isset($_POST['exit_customer'])) {
+      setcookie("GIGFILLIATE_PLACING_ORDER_FOR_CUSTOMER",null, time() - 3600,"/");
+    }
+    header('Location: ' . $_SERVER['HTTP_REFERER']);
+  }
+  public function customer_notice(){
+    if(isset($_COOKIE["GIGFILLIATE_PLACING_ORDER_FOR_CUSTOMER"])) {
+      $customer = get_user_by("ID", $_COOKIE["GIGFILLIATE_PLACING_ORDER_FOR_CUSTOMER"]);
```

Comment:
I never really manager cookies like this, I think they are easier to manage with JavaScript, but if it works thats great.

Reply:
I will have moved some code to javascript with the help of snippet you provided will move more in next commit.